### ISSUE TITLE
Add new API: modules state

### ISSUE THREAD

**Issue by ddurieux** [surveil/surveil-meta#108]
Add API status/modules to see information about the modules (broker, scheduler, external modules...)

**Comment by aviau** [comment]
Is this already available via some http API in alignak?

**Comment by ddurieux** [comment]
no, it's a Alignak module send hosts and services states in surveil
